### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by yichuandoris** [yichuandoris/MI-449-SS17-740-css-frameworks-p1twXJ#1]
@chrisvfritz Can you take a look at this? It's [hosted here](https://yichuandoris.github.io/MI-449-SS17-740-css-frameworks-p1twXJ/) and meets the following criteria:

- [x] Include the CSS and JS for Bootstrap 3 (not 4) in the page (including the "jQuery" dependency)
- [x] Use the `container` class to center your page content
- [x] Use a `navbar` class to display the name of your site and a list of at least 3 links (to past projects, or anything else) at the top of your page
- [x] Split your main content into 2 columns on large (`lg`) and medium (`md`) sized screens, but collapse them on top of each other on small (`sm`) and extra small (`xs`) screens
- [x] Use a `thumbnail` class on an at least one image in your page

**Comment by egillespie** [comment]
Hi @yichuandoris, you're off to a good start! I can clearly see the navbar and 2-column layout on my laptop. The thumbnail image looks good, too. :fist_raised: 

You might have noticed the big gray box at the very top of your page. Bootstrap calls that a "jumbotron." It's not really something we're using in this project and makes your page look a little strange. Do you mind removing it?

While you're editing the HTML, do you mind fixing the indenting? It's a little difficult to review the code when tags don't line up on the left edge. In Atom you can select all of the lines in the file and then choose `Edit` > `Lines` > `Auto Indent` to quickly re-indent the whole file.

The other thing I noticed is when I shrink the width of my window down, the thumbnail and "Progress" column switch from 6 columns to 8 columns before dropping down to the 12-column layout. The "About" `div` appears to be working correctly, though, so you can take the column styles you used there and put them on the thumbnail and "Progress" column to satisfy the 4th criteria.

Comment back here when you've made the updates and have pushed your code to GitHub. And of course, let me know if you have any questions!

**Comment by yichuandoris** [comment]
Thank you Erik! I've made the revisions based on your suggestions. Thanks a lof for letting me know the auto-indent function. Super helpful! I am not sure if I fix the final problem correctly, so please let me know if I misunderstood. Thank you. - Doris

**Comment by egillespie** [comment]
Yeah, you got it! The page looks great and the 2-column/1-column transition on different screen sizes is working wonderfully.

If you want to see how I tested the transition from 2-column to 1-column, feel free to ask in class.

:shipit:
